spyder/widgets/tabs.py

```diff
@@ -352,16 +352,34 @@ def add_corner_widgets(self, widgets, corner=Qt.TopRightCorner):
         self.set_corner_widgets({corner:
                                  self.corner_widgets.get(corner, [])+widgets})
 
+    def get_offset_pos(self, event):
+        """
+        Add offset to position event to capture the mouse cursor
+        inside a tab.
+        """
+        # This is necessary because self.tabBar().tabAt(event.pos()) is not
+        # returning the expected index. For further information see
+        # spyder-ide/spyder#12617
+        point = event.pos()
+        if sys.platform == 'darwin':
+            # The close button on tab is on the left
+            point.setX(point.x() + 3)

```

Comment:
@steff456 and @dalthviz could  you check if this works as well if we are using the high DPI scaling property to make things bigger/smaller?

Reply:
It is working for me in fullscreen mode

Reply:
@steff456, Gonzalo means if this is still working when you go to

    Preferences > General > Screen resolution

and set a custom resolution of 1.5 or 2 there.

Reply:
Works for me on Windows :+1: (although maybe should I try to test this on Mac and/or changing the OS scale instead?)

Reply:
Thanks @dalthviz for your help!